dsc/workflows/base/__init__.py

```diff
@@ -296,40 +313,17 @@ def submit_items(self, collection_handle: str) -> list:
                 self.workflow_events.errors.append(str(exception))
                 self.submission_summary["errors"] += 1
 
+                item_submission.status = ItemSubmissionStatus.SUBMIT_FAILED
+                item_submission.status_details = str(exception)
+                item_submission.submit_attempts += 1
+                item_submission.update_db()
```

Comment:
This feels good here.  Checked the `ItemSubmission` class and not seeing any surprising side effects, just trying to do things and raising errors where needed.